exercises/simple-cipher/simple_cipher_test.exs

```diff
@@ -0,0 +1,117 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("simple_cipher.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule SimpleCipherTest do
+  use ExUnit.Case
+
+  describe "random key generation" do
+    #@tag :pending
+    test "random key is all lowercase letters" do
+      key = SimpleCipher.key
+
+      assert key != nil
+      assert String.match?(key, ~r{\A[a-z]+\z}) == true
+    end
+
+    @tag :pending
+    test "random key is 100 characters long" do
+      key = SimpleCipher.key
+
+      assert String.length(key) == 100
+    end
+  end
+
+  describe "basic encoding/decoding" do
+    @tag :pending
+    test "encoding with 'a' returns the original text" do
+      assert SimpleCipher.encode("a", "a") == "a"
+      assert SimpleCipher.encode("b", "a") == "b"
+      assert SimpleCipher.encode("c", "a") == "c"
+    end
+
+    @tag :pending
+    test "encoding with another key returns shifted text" do
+      assert SimpleCipher.encode("a", "d") == "d"
+      assert SimpleCipher.encode("b", "d") == "e"
+      assert SimpleCipher.encode("c", "d") == "f"
+    end
+
+    @tag :pending
+    test "decoding with 'a' returns the original text" do
+      assert SimpleCipher.decode("a", "a") == "a"
+      assert SimpleCipher.decode("b", "a") == "b"
+      assert SimpleCipher.decode("c", "a") == "c"
+    end
+
+    @tag :pending
+    test "decoding with another key returns unshifted text" do
+      assert SimpleCipher.decode("d", "d") == "a"
+      assert SimpleCipher.decode("e", "d") == "b"
+      assert SimpleCipher.decode("f", "d") == "c"
+    end
+
+    @tag :pending
+    test "key uses per-letter translation for encoding" do
+      key = "abc"
+
+      assert SimpleCipher.encode("abc", key) == "ace"
+      assert SimpleCipher.encode("bcd", key) == "bdf"
+      assert SimpleCipher.encode("cde", key) == "ceg"
+      assert SimpleCipher.encode("iamapandabear", "dddddddddddddd") == "ldpdsdqgdehdu"
+    end
+
+    @tag :pending
+    test "key uses per-letter translation for decoding" do
+      key = "abc"
+
+      assert SimpleCipher.decode("ace", key) == "abc"
+      assert SimpleCipher.decode("bdf", key) == "bcd"
+      assert SimpleCipher.decode("ceg", key) == "cde"
+      assert SimpleCipher.decode("ldpdsdqgdehdu", "dddddddddddddd") == "iamapandabear"
+    end
+
+    @tag :pending
+    test "only lowercase a-z are translated, rest are passed through" do
+      assert SimpleCipher.encode("this is a test!", "d") == "wklv lv d whvw!"
+      assert SimpleCipher.decode("wklv lv d whvw!", "d") == "this is a test!"
+    end
+
+    @tag :pending
+    test "decoding an encoded text with the same key returns original text" do
```

Comment:
You're right, I can't think of one. We've already implicitly proven in almost every previous test that `text |> encode(key) |> decode(key) == text`, so this holdover from the Ruby version is unnecessary.